### PR TITLE
Feature/web/126 create new players on execution page

### LINKED ISSUE #126: Create new players on Execution page
There should be a form to create new players on the Execution page. This includes generating a TAN and corresponding QR-Code.

### LINKED ISSUE #129: Only send relevant information in polling requests
Currently all information about the execution is send back on each poll (every 5 seconds). This can be reduced by filtering out information that does not change.

### PR DESCRIPTION
**Pull request by 7erra**
This PR adds the following features:
- Ability to create new players on the Execution page
- Creates and saves new executions from and to the database

Closes #126, closes #129

### CODE REVIEW

File: server/execution/services/entityloader.py [hunk 1]
```diff
@@ -185,3 +194,10 @@ def load_execution(exec_id: int) -> bool:
         # Activate execution (makes it accessible by API)
         run.activate_execution(execution)
         return True
+
+
+def load_role(id: int) -> Role | None:
```
Comment (YIllmann) [inline server/execution/services/entityloader.py:199]: is this maybe a duplicate to 139?

Comment (7erra) [reply server/execution/services/entityloader.py:199]: 139?

Comment (YIllmann) [reply server/execution/services/entityloader.py:199]: line 139

File: server/execution/entities/execution.py [hunk 2]
```diff
@@ -49,3 +51,17 @@ def to_json(self, shallow: bool = False):
         only the object reference in form of a unique identifier is included.
         """
         return json.dumps(self.to_dict(shallow))
+
+    def add_new_player(self, role: int, location: int):
+        from utils.tans import uniques
+        from execution.run import register_player
+        from execution.services.entityloader import load_location
+        from execution.services.entityloader import load_role
+        tan = str(uniques(1)[0])
```
Comment (simohlsen) [inline server/execution/entities/execution.py:60]: Doesn't `unqiues` return strings already?

And do you mind, if I refactor this later and create an additional method in the Tan-class that returns a single unique TAN, as this is probably the most-likely use case?

**Review by simohlsen — APPROVED** [review]

File: server/execution/services/entityloader.py (hunk 1 above)
Comment (7erra) [reply server/execution/services/entityloader.py:199]: Ah true, I will make that function public

File: server/execution/entities/execution.py (hunk 2 above)
Comment (simohlsen) [reply server/execution/entities/execution.py:60]: Nevermind, returns TANs obviously 😅

Comment (7erra) [reply server/execution/entities/execution.py:60]: Nope, uniques returns a list of Tans (the object). And no, I don't mind as I do have missed that functionality myself :)

**Review by YIllmann — APPROVED** [review]

**Review by simohlsen — APPROVED** [review]

**Pull request merged**